### ISSUE TITLE
Install name for pip

### ISSUE THREAD

**Issue by HeikoSchuett** [rsagroup/rsatoolbox#173]
A discussion point on the name:

we currently need to run
``` pip install rsa3 ```
to install our toolbox as the name pyrsa is taken by a python implementation of the RSA encryption algorithm. 

Unfortunately, this has bad consequences. For example, I managed to get into a state where I cannot successfully install our toolbox anymore after running 
```
conda create -n test_pip
pip install pyrsa 
python 
import pyrsa
# wrong package
exit()
pip install rsa3
python 
import pyrsa
 # does not work
exit()
pip uninstall pyrsa
python
import pyrsa
# does not work
exit()
```
i.e. once I installed the two into the same place once the environment is broken permanently. 

Of course none of this is supposed to happen. Do we know of a way to fix this? Or should we do a full rename soon, such that we do not run into such issues anymore?

**Comment by JasperVanDenBosch** [comment]
You mean rename everything to rsa3?

**Comment by JasperVanDenBosch** [comment]
Perhaps that's the best approach yes.

**Comment by HeikoSchuett** [comment]
Yes, renaming everything to rsa3 (or something different) is my proposal

**Comment by JasperVanDenBosch** [comment]
fixed in #189
